### DISCUSSION TITLE
How can I stretch textfield remaining space

### DISCUSSION THREAD

**Discussion by heejit** [cogentcore/core#1221] [General]
I want to move button to right side at the end of window and textfield to stretch till the button

I have set following 

```
vFrame1 := core.NewFrame(this.Body)
	vFrame1.Styler(func(s *styles.Style) {
		s.Grow.Set(1, 0)
	})

this.TextOptions = core.NewTextField(vFrame1).SetPlaceholder("Enter any data required by selected action")
	this.TextOptions.Styler(func(s *styles.Style) {
		s.Grow.Set(1, 0)
	})

```
![image](https://github.com/user-attachments/assets/87def099-6998-4932-bf52-c97c2051a80d)

**Comment by kkoreilly** [comment]
Try `s.Max.Zero()` in the `TextField` `Styler`.

**Reply by heejit** [reply]
thanks working.
